charts/assets/Chart.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: assets
+description: A Helm chart for Kubernetes
+type: application
+version: 0.0.1
```

We should use the same versioning that we have in the rest of charts, starting with `v0.1.0` (first feature). In semantic versioning 0.0.1 means "no features, but a bug patched"
```suggestion
version: v0.1.0
```